lib/css/components/_stacks-prose.less

```
@@ -280,6 +281,7 @@
         border-radius: @br-md;
         color: var(--black-800);
         cursor: pointer;
+        min-height: 48px;
```

```suggestion
        min-height: @su48; // TODO: Let's find a solution that doesn't have a magic number
```

Dang ol' magic numbers. Don't have the energy to look into a way of avoiding this right now, ship it.